src/runner/Resources.resx

```diff
@@ -89,7 +148,7 @@
     <value>This setting has been disabled by your administrator.</value>
   </data>
   <data name="STARTUP_DISABLED_BY_USER" xml:space="preserve">
-    <value>This setting has been disabled manually via &lt;a href=&quot;https://ms_settings_startupapps&quot; target=&quot;_blank&quot;&gt;Startup Settings&lt;/a&gt;.</value>
+    <value>This setting has been disabled manually via &lt;a href="https://ms_settings_startupapps" target="_blank"&gt;Startup Settings&lt;/a&gt;.</value>
```

Comment:
perhaps use `<![CDATA[` to avoid too many escapes.

Reply:
This should perhaps be done for all resource files. This was not a change I made however, and I do not know the decision process behind this. Perhaps another PR should be opened to fix all these occurences?